components/engine_traits/src/errors.rs

```diff
@@ -149,6 +149,8 @@ pub enum Error {
     EntriesUnavailable,
     #[error("The entries of region is compacted")]
     EntriesCompacted,
+    #[error("The region is not available in the region cache engine")]
+    CacheNotAvailable,
```

Comment:
there are some reason that causes the cache miss:
1. read_ts is too old 
2. cache is not ready 
so, Do you need to record the failed reason?

Reply:
this error is intended to represent that the boundary of IterOptions is not set. I have adjusted the name.